app/controllers/mixins/generic_show_mixin.rb

```diff
@@ -128,7 +148,7 @@ def nested_list(table_name, model, options = {})
       drop_breadcrumb(:name => _("%{name} (All %{title})") % {:name => @record.name, :title => title},
                       :url  => show_link(@record, :display => @display))
 
-      view_options = {:parent => @record}
+      view_options = {:parent => @record, :association => options[:association]}
```

Comment:
One line below I did the same, but tested for presence of the key. I did not dig deeper into `get_view` to see if there's a difference between the key not being passed and the value being `nil`.
But no matter which variant you choose, I'd say that we should use the same pattern on both lines.